### ISSUE TITLE
CCC IDs vs. Clinic IDs

### ISSUE THREAD

**Issue by chris-githu** [chap-uzima/IQCare#63]
@vogoti 
A significant number of patients in `Njabini` and possible other sites only had `Clinic IDs` and no `Patient Enrollment IDs` in v4
421 migrated these successfully but IQCare KHMIS expects that all CCC clients have a 10-digit CCC ID.
Please discuss internally what we should do about this as it may create issues in the future.

FYI @lmwaniki @mkaburu

**Comment by vogoti** [comment]
@chris-githu True,Njabini is most affected.
Can we be able to do a data cleanup from the frontend on this?How can we handle this for non migrated sites?
@mkaburu @jkarigi @lmwaniki

**Comment by chris-githu** [comment]
@vogoti How about we re-migrate the Clinic IDs as CCC IDs (for patients missing a CCC ID)

**Comment by chris-githu** [comment]
@vogoti 
What was the decision on this?

**Comment by vogoti** [comment]
@chris-githu Yes,i agree if we can remigrate the IDs it would be better than the facility updating one by one.FYI @jkarigi

**Comment by chris-githu** [comment]
@vogoti @jkarigi 
Ok. Let's do that. I will share the script via email
